### PR TITLE
Update LearnSidebar.ejs - add Svelte's links

### PR DESCRIPTION
**Pull request by RafaelGreen1**

### CODE REVIEW

File: macros/LearnSidebar.ejs [hunk 1]
```diff
@@ -2032,6 +2095,21 @@ var text = mdn.localStringMap({
         </ol>
     </details>
   </li>
+  <li class="toggle">
+    <details <%=currentPageIsUnder('Tools_and_testing/Client-side_JavaScript_frameworks')%>>
+        <summary><%=text['Svelte']%></summary>
+        <ol>
+          <li><a href="<%=baseURL%>Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_getting_started"><%=text['Getting_started_with_Svelte']%></a></li>
+          <li><a href="<%=baseURL%>Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_Todo_list_beginning"><%=text['Starting_our_Svelte_Todo_list_app']%></a></li>
+          <li><a href="<%=baseURL%>Tools_and_testing/Client-side_JavaScript_frameworks/Svelte_variables_props"><%=text['Dynamic_behavior_in_Svelte']%></a></li>
```
Comment (chrisdavidmills) [inline macros/LearnSidebar.ejs:2104]: Apologies for taking so long to review this. This PR looks mostly great; the only change I can see that is needed is that the above line has the wrong text reference. It should be `Dynamic_behavior_in_Svelte:_working_with_variables_and_props`.

Once that is fixed, I think we can merge this.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Only one small change request; thanks @RafaelGreen1 !

**Review by chrisdavidmills — APPROVED** [review]
Looks perfect, thanks @RafaelGreen1 . Let's merge this.

**Comment by RafaelGreen1** [comment]
for some reason, the svelte links are still not showing in the sidebar
@chrisdavidmills do you have any why?
Thanks,

**Comment by chrisdavidmills** [comment]
Hi @RafaelGreen1 — the sidebar update won't be live as soon as the merge happens. A product push is required, and then the pages that feature this sidebar need to be regenerated. This usually happens once per week, to my knowledge, so we should see the changes live by the end of this week, or early next week.

**Comment by escattone** [comment]
@RafaelGreen1 Just FYI, this change went "live" on MDN at 12:38pm Pacific Time today. As @chrisdavidmills said, you may still have to re-render the pages that use this sidebar (log into MDN, go to the Wiki version of the page you want to re-render, and then hit shift-key+reload). Cheers!

**Pull request merged**